pkg/datastore/datastore.go

```diff
@@ -28,15 +28,25 @@ func New() *memoryDatastore {
 
 // Reader is the interface that wraps the Read method.
 type Reader interface {
-	// Read takes a package identifier and returns metadata defined in the opsrc
+	// Read takes an OperatorSource name and package identifier and returns metadata defined in the opsrc
 	// spec to associate that package to a specific opsrc in the marketplace.
 	// The OpsrcRef returned can be used to determine how to download manifests
 	// for a specific operator package.
-	Read(packageID string) (opsrcMeta *OpsrcRef, err error)
+	Read(source, packageID string) (opsrcMeta *OpsrcRef, err error)
 
-	// ReadVersion takes a package identifer and returns version metadata
+	// ReadByPackageName takes a package identifier and returns metadata defined in the opsrc
+	// spec to associate that package to a specific opsrc in the marketplace.
+	// The OpsrcRef returned can be used to determine how to download manifests
+	// for a specific operator package.
```

Comment:
We should call out that what happens if the package is present in multiple `OperatorSources`.

Reply:
Sure thing!